### PR TITLE
GitHub actions CI

### PR DESCRIPTION
**Pull request by fneum**
Building times comparison (minutes)

|         | Travis | GH Actions |
|---------|--------|------------|
| ubuntu  | 7      | 7          |
| macos   | 12     | 8          |
| windows | 21     | 16         |

### CODE REVIEW

File: test/config.test1.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ countries: ['DE']
 
 snapshots:
   start: "2013-03-01"
-  end: "2014-04-01"
+  end: "2013-03-08"
```
Comment (FabianHofmann) [inline test/config.test1.yaml:23]: that's intended? (guess it makes sense, since we just want to test the code)

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      uses: conda-incubator/setup-miniconda@v2.1.1 
+      with: # checks out environment 'test' by default
+          mamba-version: "*"
+          channels: conda-forge,defaults
+          channel-priority: true
+
+    - name: Install dependencies  
+      run: |
+        echo -ne "url: ${CDSAPI_URL}\nkey: ${CDSAPI_TOKEN}\n" > ~/.cdsapirc
+        echo -e "  - glpk\n  - ipopt<3.13.3" >> envs/environment.yaml
```
Comment (FabianHofmann) [inline .github/workflows/ci.yaml:39]: perhaps in the long term we should add these to the default env?

**Review by FabianHofmann — APPROVED** [review]
Looks great!

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (fneum) [reply .github/workflows/ci.yaml:39]: Maybe, so far we have kept the solvers out of the dependencies.

File: test/config.test1.yaml (hunk 1 above)
Comment (fneum) [reply test/config.test1.yaml:23]: Yes, shaves about 20 seconds off the runtime.

**Pull request merged**